src/client/src/mockData/mockVsCodeApi.ts

```diff
@@ -84,7 +84,20 @@ const mockVsCodeApi = () => ({
                 locations: [{ label: WEST_US, value: WEST_US }],
                 resourceGroups: [
                   { label: RESOURCE_GROUP_MOCK, value: RESOURCE_GROUP_MOCK },
```

Comment:
should this label and value be changed so that it is consistent with the rest?

Reply:
So for this one, we could just change `RESOURCE_GROUP_MOCK` to hardcoded string, or we could create more const variables at the top like here:
https://github.com/microsoft/WebTemplateStudio/blob/297d2537b4328eaedb9e5913228d4fed14caf6a8/src/client/src/mockData/mockVsCodeApi.ts#L8

Should I just make more const variables?